nestkernel/random_generators.h

```diff
@@ -266,10 +273,18 @@ class RandomGenerator final : public BaseRandomGenerator
     return uniform_ulong_dist_( rng_, param );
   }
 
+  inline unsigned long
+  prand( double mu ) override
+  {
+    std::poisson_distribution< unsigned long >::param_type param( mu );
+    return poisson_ulong_dist_( rng_, param );
+  }
+
 private:
   RandomEngineT rng_; //!< Wrapped RNG engine.
   std::uniform_int_distribution< unsigned long > uniform_ulong_dist_;
   std::uniform_real_distribution<> uniform_double_dist_0_1_;
+  std::poisson_distribution< unsigned long > poisson_ulong_dist_;
```

Comment:
See above

Reply:
Addressed in 29f6d751013b363b90ed241b363c7205d37cfb60.